client/src/modules/Schedule/components/FilteredTags/FilteredTags.test.tsx

```diff
@@ -0,0 +1,61 @@
+import { fireEvent, render, screen, waitFor, within } from '@testing-library/react';
+import { FilteredTags } from './FilteredTags';
+import { CourseScheduleItemDto, CourseScheduleItemDtoTagEnum } from 'api';
+import { TAG_NAME_MAP } from 'modules/Schedule/constants';
+
+const TagsEnum = CourseScheduleItemDtoTagEnum;
+
+describe('FilteredTags', () => {
+  const onTagCloseMock = jest.fn();
+
+  it('should not render when tags were not provided', () => {
+    render(<FilteredTags tagFilter={[]} onTagClose={onTagCloseMock} />);
+
+    expect(screen.queryByText(/Tag: /)).not.toBeInTheDocument();
+  });
+
+  it.each`
+    tag
+    ${TagsEnum.Coding}
+    ${TagsEnum.CrossCheck}
+    ${TagsEnum.Interview}
+    ${TagsEnum.Lecture}
+    ${TagsEnum.SelfStudy}
+    ${TagsEnum.Test}
+  `('should render tag "$tag"', ({ tag }: { tag: CourseScheduleItemDto['tag'] }) => {
+    render(<FilteredTags tagFilter={[tag]} onTagClose={onTagCloseMock} />);
+
+    expect(screen.getByText(getTagLabel(tag))).toBeInTheDocument();
+  });
+
+  it('should render several tags', () => {
+    render(
+      <FilteredTags
+        tagFilter={[TagsEnum.Coding, TagsEnum.CrossCheck, TagsEnum.Interview]}
+        onTagClose={onTagCloseMock}
+      />,
+    );
+
+    expect(screen.getAllByText(/Tag: /)).toHaveLength(3);
+  });
+
+  it('should remove selected tag when onTagClose was called', () => {
+    render(
+      <FilteredTags
+        tagFilter={[TagsEnum.Coding, TagsEnum.CrossCheck, TagsEnum.Interview]}
+        onTagClose={onTagCloseMock}
+      />,
+    );
+    const interviewTag = screen.getByText(getTagLabel(TagsEnum.Interview));
+    const interviewCrossIcon = within(interviewTag).getByLabelText('close');
+
+    fireEvent.click(interviewCrossIcon);
+
+    expect(onTagCloseMock).toHaveBeenCalled();
+    expect(onTagCloseMock).toHaveBeenCalledWith(TagsEnum.Interview);
```

Comment:
Minor: `toHaveBeenCalledWith` also means that `onTagCloseMock` was called, isn't it? I think we could leave only line 55 here

Reply:
Agree. Fixed